rotkehlchen/exchanges/coinbase.py

```diff
@@ -535,8 +535,10 @@ def _query_single_account_transactions(
                         f'Transaction of type trade doesnt have the '
                         f'expected structure {transaction}',
                     )
-            elif tx_type in ('buy', 'sell'):
-                self._process_normal_trade(event=transaction, trades=trades)
+            elif tx_type in ('buy', 'sell', 'advanced_trade_fill'):
+                trade = self._process_coinbase_trade(event=transaction)
+                if trade:
+                    trades.append(trade)
```

Comment:
a bit of a nitpick but this can be done now in 3.11
```suggestion
                if (trade := self._process_coinbase_trade(event=transaction)):
                    trades.append(trade)
```